build_library/build_image_util.sh

```diff
@@ -506,6 +506,16 @@ EOF
         "${BUILD_DIR}/${image_kconfig}"
   fi
 
+  # Build the selinux policy and apply file labels.
+  if pkg_use_enabled coreos-base/coreos selinux; then
+    setup_qemu_static "${root_fs_dir}"
+    sudo chroot "${root_fs_dir}" /bin/bash -x << 'EOF'
+      (cd /usr/share/selinux/mcs && semodule -s mcs -i *.pp)
+      setfiles -F /usr/lib/selinux/mcs/contexts/files/file_contexts /usr/lib/modules
```

Comment:
What is special about the kernel modules that require relabeling?

Reply:
We build up the board's rootfs '${root_fs_dir}' from the SDK host, so the rootfs files don't have any selinux labels. The kernel module loader (modprobe, etc.) checks for proper labels, so without the relabeling here module loading will fail when selinux is enforcing.  I didn't look into why module loading worked with the old selinux policy/tools.